controllers/graph.controller.js

```diff
@@ -0,0 +1,84 @@
+const graphDbService = require("../services/graph.service");
+const {
+  ERROR_GRAPH_NOT_FOUND,
+  ERROR_INVALID_GRAPH,
+  INTERNAL_ERROR,
+} = require("../constants");
+const { isValidGraph } = require("../services/utils");
+
+const publishGraph = async (req, res) => {
+  try {
+    const { adjacencyLists, userId } = req.body;
+    if (!isValidGraph(adjacencyLists)) {
+      res.send({ error: ERROR_INVALID_GRAPH });
+      return;
+    }
+    const newGraph = await graphDbService.DbCreateGraph({
+      adjacencyLists,
+      userId,
+    });
+    return res.send(newGraph);
+  } catch (err) {
+    console.log(err);
+    res.send({ error: INTERNAL_ERROR });
+  }
+};
+
+const getGraphById = async (req, res) => {
+  try {
+    const { id } = req.body;
+    const tryGetGraph = await graphDbService.DbReadGraph(id);
+    if (tryGetGraph) {
+      res.send(tryGetGraph);
+      return;
+    } else {
```

Comment:
Change this line into `return res.send(tryGetGraph)` just to be consistent. 

Also `tryGetGraph` sounds like a method name. 

Change this into `graph`,  also the `Db` in your function name could be removed so you will be stay with `readGraph`